src/algebra/big_operators/basic.lean

```diff
@@ -1276,6 +1276,34 @@ end comm_group_with_zero
 
 end finset
 
+namespace fintype
+
+open finset
+
+/-- `fintype.prod_bijective` is a variant of `finset.prod_bij` that accepts `function.bijective`. -/
+@[to_additive "`fintype.sum_equiv` is a variant of `finset.sum_bij` that accepts
+`function.bijective`"]
```

Comment:
```suggestion
/-- `fintype.prod_bijective` is a variant of `finset.prod_bij` that accepts `function.bijective`.

See `function.bijective.prod_comp` for a version without `h`. -/
@[to_additive "`fintype.sum_equiv` is a variant of `finset.sum_bij` that accepts
`function.bijective`.

See `function.bijective.sum_comp` for a version without `h`. "]
```

Reply:
We already have `function.bijective.prod_comp`, do we need one more version?

Reply:
It sounds like the presence of the `h` argument makes the lemma a lot more usable, as demonstrated by the rest of the diff